packages/editor/src/browser/editor-command.ts

```diff
@@ -150,6 +150,14 @@ export namespace EditorCommands {
         category: CommonCommands.VIEW_CATEGORY,
         label: 'Toggle Word Wrap'
     });
+    /**
+     * Command that toggles sticky scroll.
+     */
+     export const TOGGLE_STICKY_SCROLL = Command.toDefaultLocalizedCommand({
+        id: 'editor.action.toggleStickyScroll',
+        category: CommonCommands.VIEW_CATEGORY,
+        label: 'Toggle Sticky Scroll'
+    });
```

Comment:
There is no default localization present for "toggle sticky scroll" under https://github.com/eclipse-theia/theia/blob/master/packages/core/src/common/i18n/nls.metadata.json which would permit us to use `toDefaultLocalizedCommand`. Instead we need to provide our custom one (which will get generated before the next release):

```suggestion
     export const TOGGLE_STICKY_SCROLL = Command.toLocalizedCommand({
        id: 'editor.action.toggleStickyScroll',
        category: CommonCommands.VIEW_CATEGORY,
        label: 'Toggle Sticky Scroll'
    }, 'theia/editor/toggleStickyScroll', EDITOR_CATEGORY_KEY);
```

Reply:
many thanks for your suggestion, correction was done.